servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImpl.java

```diff
@@ -3674,6 +3674,7 @@ void setRoleMemberExpiration(final AthenzDomain domain, final Role role, final R
 
         boolean bUser = ZMSUtils.isUserDomainPrincipal(roleMember.getMemberName(), userDomainPrefix,
                 addlUserCheckDomainPrefixList);
+        boolean bGroup = roleMember.getMemberName().contains(AuthorityConsts.GROUP_SEP);
```

Comment:
Looking at the setRoleMemberExpiration function, it is called from 2 places and in both cases the roleMember object that is passed already includes the type since we call:         roleMember.setPrincipalType(principalType(roleMember.getMemberName()));

So rather than determining the bUser and bGroup values again, we already know the type so we should remove those two lines and convert this into a simple case statement - similar to validateRoleMemberPrincipal method.